NEWS.md

```diff
@@ -1,5 +1,7 @@
 # httr2 (development version)
 
+* `req_perform_stream()` has been soft deprecated.
+* Deprecated functions `mutli_req_perform()`, `req_stream()`, `with_mock()` and `local_mock()` hav been removed. Deprecated arguments `req_perform_parallel(pool)` and `req_oauth_auth_code(host_name, host_ip, port)`/ `oauth_flow_auth_code()` have been removed.
```

Comment:
```suggestion
* Deprecated functions `mutli_req_perform()`, `req_stream()`, `with_mock()` and `local_mock()` hav been removed. Deprecated arguments `req_perform_parallel(pool)` and `req_oauth_auth_code(host_name, host_ip, port)`/ `oauth_flow_auth_code(host_name, host_ip, port)` have been removed.
```
I misread this at first and thought you'd removed `oauth_flow_auth_code()` entirely (even though in retrospect that's clearly not what you meant).